### ISSUE TITLE
Windows Authentication Broken

### ISSUE THREAD

**Issue by darl0026** [a85/POSTMan-Chrome-Extension#2697]
Postman for Windows
Version 4.9.3
win32 6.1.7601 / x64

Windows authentication no longer works in this version.  When i make a call in chrome it works, but the same url does not work in Postman because the authentication is broken.  This was not the case in previous versions

**Comment by sdnts** [comment]
@darl0026 Which auth spec are you using? (Basic, Digest etc.)
Also, could you elaborate a little on what seems broken? Is it the signature?

**Comment by ehrlich-b** [comment]
Not darl, but I am having the same issue. In Postman for Chrome you can make NTLM requests with "Authorization = No Auth". Chrome automatically does NTLM authentication based on your configured internet options. 

This doesn't seem to work in Postman for Windows, making it basically unusable for us.

**Comment by somhpq** [comment]
I am a king of new - I have Postman for Windows 64 - version 5.0.0 at work.

1. It does not work with NTLM - I set the Authorization Type to "No Auth" - with no Headers
2. It does not work with NTLM - I set the Authorization Type to "No Auth" - with Headers of Authorization : Negotiate xxxxxxx ...... ( I stole the value from Chrome Dev Tool - Chrome browser can hit the GET action directly )
3. I do not have Postman for Chrome, they do not allow so, they love to stick with IE

My questions
**- do I need any other download to have NTLM Auth work ? (per the #266 from @kunagpal)**
**- will it work when setting Authorization to "No Auth" or it will have NTLM or Negotiate as an option along with other Authorization methods ?**

Thanks a lot !
